shop/views.py

```diff
@@ -36,7 +47,8 @@ def product_view(request, pk):
     return render(request, 'shop/detail.html', {
         'product': product,
         'product_rating': product_rating,
-        'positive_rating_percentage': positive_rating_percentage
+        'positive_rating_percentage': positive_rating_percentage,
+        'comments': comments
```

Comment:
Inget att påpeka riktigt, men när man har listor eller dylikt och lägger till en ny rad i slutet brukar man lägga in ett komma-tecken direkt så det inte kommer up som en diff i commiten.